src/Arcus.Security.Core/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Arcus.Security.AzureFunctions")]
```

Comment:
What does this do? 🤔

Reply:
Yes, was not entirely sure if this is the correct way. I need the `SecretStoreBuilder.RegisterProviders()` internal method in the extension written in the Azure Functions project. This was made internal bc it isn't recommanded to expose it IMHO. But that leaves us with a problem here.
We could introduce a separate model where we call this method, and make that one public, or use `InternalsVisibleTo`. Or another proposal from you?